utils/help_menus

```diff
@@ -1699,6 +1699,11 @@ generate_ui_web_doc() {
     fi
     echo "Documentation downloaded successfully to $output_file"
 
+    echo "Patching the title"
+    gruim_version=$(kubectl get secret -n grpl-system grsf-config -o jsonpath="{.data.uiimagetag}" | base64 -d)
```

Comment:
hi @patrick-grapple 
sed doesn't work same on mac linux
when I ran this code on linux if get following error

![Screenshot from 2024-11-28 10-35-57](https://github.com/user-attachments/assets/c485ae59-89b9-43f0-8f97-21e39c46230e)

So I updated the code to use awk instead

![Screenshot from 2024-11-28 10-46-40](https://github.com/user-attachments/assets/083f21d7-e9c9-47e5-81d3-cbc608c15670)
![Screenshot from 2024-11-28 10-47-19](https://github.com/user-attachments/assets/3b05376b-e99b-48b5-b151-59ef4be1f1bc)

After this change the code worked fine and title was updated successfully on UI

Reply:
thanks a lot! 👍